src/components/button/button.styles.ts

```diff
@@ -66,7 +66,7 @@ export default css`
     display: inline-block;
   }
 
-  .button__label::slotted(sl-icon) {
+  .button__label::slotted(.button__caret) {
```

Comment:
No harm in this change imo, makes it flexible at literally no downside and using classes here is a good practice anyways.